ui/src/admin/components/chronograf/AllUsersTableRowNew.js

```diff
@@ -36,15 +37,13 @@ class AllUsersTableRowNew extends Component {
   handleConfirmCreateUser = () => {
     const {onBlur, onCreateUser} = this.props
     const {name, provider, scheme, roles, superAdmin} = this.state
-
     const newUser = {
       name,
       provider,
       scheme,
       superAdmin,
-      roles: roles[0].id === null ? [] : roles,
+      roles: roles[0].organization === undefined ? [] : roles,
```

Comment:
consider comparing roles to nullrole?

Reply:
consider sending `roles: [role]`  and dealing with a `role` var in the rest of this file.

Reply:
For comparing to null roles, it was returning false. Despite the content of the role being the same, the comparison was returning false.